src/DotNetWorker.Core/ExtensionStartup/WorkerExtensionStartupAttribute.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Core
+{
+    /// <summary>
+    /// An attribute to inform that this assembly contains a worker extension startup type.
+    /// </summary>
+
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = true)]
+    public class WorkerExtensionStartupAttribute : Attribute
```

Comment:
Extension authors should use this attribute to decorate the assembly in which they have their implementation of `IWorkerExtensionStartup`.